tensorflow/cc/gradients/math_grad_test.cc

```diff
@@ -865,5 +865,13 @@ TEST_F(NaryGradTest, Minimum) {
   RunTest(x, x_init_value, y, shape);
 }
 
+TEST_F(NaryGradTest, Select) {
+  TensorShape shape({3, 4});
+  auto x1 = Placeholder(scope_, DT_FLOAT, Placeholder::Shape(shape));
+  auto x2 = Placeholder(scope_, DT_FLOAT, Placeholder::Shape(shape));
+  auto y = Where3(scope_, Greater(scope_, x1, x2), x1, x2);
```

Comment:
@gunan @drpngx There seems something wrong with the condition `x1 > x2`, which might be unstable for gradient calculation.

Reply:
@drpngx I want to confirm that:  Is `NaryGradTest.Select` the only one failed test?

Reply:
I can't remember on the top of my head. I thought there was another one. You can try `bazel test --runs_per_test=100` and you should find it.

Reply:
Fine, I'll look into it later. Thanks for your quick reply, @drpngx .

Reply:
I have opened #15807 to fix the unstable test case. Might you take a look? Thanks.